docs/cmd_list.txt

```diff
@@ -8,32 +8,30 @@ Options:
   --help                          Show this message and exit.
 
 Configuration commands:
-  ping  Health check REANA server.
+  ping  Check connection to REANA server.
 
 Workflow management commands:
-  create  Create a REANA compatible workflow from REANA specifications file.
-  delete  Delete a workflow run.
-  diff    Show differences between two workflows.
-  list    List all available workflows.
+  create  Create a new analysis workflow.
+  delete  Delete a workflow.
+  diff    Show diff between two worklows.
+  list    List all user's workflows.
 
 Workflow execution commands:
   logs      Get workflow logs.
-  run       Create, upload and start the REANA workflow.
+  run       Create, upload and start a new analysis workflow.
   start     Start previously created workflow.
-  status    Get status of a previously created workflow.
-  stop      Stop a running workflow
-  validate  Validate the REANA specification.
+  status    Get status of previously created workflow.
+  stop      Stop a running workflow.
+  validate  Validate workflow specification file.
 
 Workspace interactive commands:
-  close  Close an interactive workflow session
-  open   Open an interactive session inside the workflow workspace
+  close  Close an interactive workflow session.
```

Comment:
"interactive session", not "interactive workflow session".  Let's use "workflows" separately from "sessions"